src/families/stellar/api/horizon.js

```diff
@@ -16,6 +16,15 @@ const currency = getCryptoCurrencyById("stellar");
 
 const server = new StellarSdk.Server(getEnv("API_STELLAR_HORIZON"));
 
+StellarSdk.HorizonAxiosClient.interceptors.response.use((x) => {
+  // FIXME: workaround for the Stellar SDK not using the correct URL: the "next" URL
+  // included in server responses points to the node itself instead of our reverse proxy...
+  const next = new URL(x.data._links.next.href);
```

Comment:
can you add a condition on the existence of this field ?
With "?." on all parts